rules/camunda-cloud/duplicate-execution-listeners.js

```diff
@@ -0,0 +1,81 @@
+const {
+  ERROR_TYPES,
+  findExtensionElement
+} = require('../utils/element');
+
+const { reportErrors } = require('../utils/reporter');
+
+const { skipInNonExecutableProcess } = require('../utils/rule');
+
+module.exports = skipInNonExecutableProcess(function() {
+  function check(node, reporter) {
+    const executionListeners = findExtensionElement(node, 'zeebe:ExecutionListeners');
+
+    if (!executionListeners) {
+      return;
+    }
+
+    const errors = hasDuplicatedExecutionListeners(executionListeners, node);
```

Comment:
I guess it's because duplication requires two duplicate properties instead of just one. We could still adjust the helper to allow multiple properties.

Reply:
I have considered this and decided against it due to the adjustment needed. If we have more cases like this, we may abstract it away, but right now I'd consider it YAGNI.

Reply:
I'll try that.

Reply:
@philippfromme please have a look now.